src/ts/features/livePreview.tsx

```diff
@@ -89,12 +94,13 @@ class PreviewIframe {
         document.addEventListener('mouseover', this.mouseOverListener)
         document.addEventListener('mouseout', this.mouseOutListener)
     }
+
     private destoryMouseListeners() {
         document.removeEventListener('mouseover', this.mouseOverListener)
         document.removeEventListener('mouseout', this.mouseOutListener)
     }
 
-    private mouseOverListener = (e: Event) => {
+    private mouseOverListener = async (e: Event) => {
```

Comment:
Does this need to be async? (note that we are also using this instance of the listener to remove, in `destroy`. Not sure what `async` will compile to in production code. It's possible we lose the reference) @Stvad

Reply:
um, why would marking function async make us loose the reference to it?   
it's not gonna compile to anything different `async` is a js keyword

Reply:
and function needs to be async to call other async functions

Reply:
um, actually scratch that. it needs to be async to use `await` so probably no for this one

Reply:
Well, `async/await` is an ES6 feature, so it should compile to `regeneratorRuntime`. I think the typescript loader would be doing that in this case.
It can't be directly used in all browsers. If we want it to not compile to anything else, we might need to add a `browserlistrc` to the webpack config, to only support chrome 55+ & FF52+. @Stvad

Reply:
For eg., see this complied code from the create-block-demo. No async/await, it uses generators.

![image](https://user-images.githubusercontent.com/2976363/82175245-11b77f00-98f1-11ea-9804-843d78248d61.png)

Reply:
we do actually compile to ES6 https://github.com/roam-unofficial/roam-toolkit/blob/master/tsconfig.json#L5, but async/await is later it seems

Reply:
We use ES2015. Async/await is  ES2017.

es6 is just the module system

Reply:
es6 ~=es2015 from what I read. anyhow, removed async here, even though I would've been really surprised if it were to make the difference :)

Reply:
It does get a little confusing, but the ECMA officially calls it `ES2015` - that's the name used in configs here too;

`ES6` is  used to refer to the module system -- which was released with `ES2015`, which was there's so much confusion in the first place -- there's no ES7,8,9, but we do have ES2017 etc :)
Now there's ESNext though